srcpkgs/sway/INSTALL

```diff
@@ -0,0 +1,5 @@
+case "${ACTION}" in
+post)
+	setcap CAP_SYS_NICE=+ep usr/bin/sway
```

Comment:
Doing this currently breaks loading custom keymaps from a user directory, so it should probably be removed and added back when https://github.com/swaywm/sway/pull/7326 is included in a future release (also see https://github.com/xkbcommon/libxkbcommon/pull/312).

Reply:
Since this has been merged in, could you post an issue for this?

Reply:
I'll just open a PR when I get the time. :)

Reply:
#41358